components/Image/image.css

```diff
@@ -0,0 +1,18 @@
+/* Normal Image */
+.img {
+  object-fit: cover;
+  height: 15rem;
+  width: 15rem;
+  border-radius: var(--border-radius-3);
+}
+
+/* Rounded Image */
+.img.img-round {
+  border-radius: var(--border-rounded);
+}
+
+/* Responsive Image */
+.img.img-responsive {
+  width: 100%;
+  height: auto;
```

Comment:
Just a suggestion. use object-fit:cover to maintain the aspect ratio according to changing dimensions.

Reply:
yeah it's getting applied to the `.img` class by default